secrets/spring-cloud-vault/README.md

```diff
@@ -55,3 +56,61 @@ $ curl -i -X DELETE http://localhost:8080/api/orders
     Content-Length: 0
     Date: Fri, 13 Apr 2018 21:50:43 GMT
 ```
+
+## Refreshing Static Secrets
+Spring has an actuator we can use to facilitate the rotation of static credentials. Example below.
+
+1. Create the old secret.
+```
+$ curl -i  \
+    --header "X-Vault-Token: root" \
```

Comment:
I'd change `root` to `${VAULT_TOKEN}` and have `$ export VAULT_TOKEN=root` above, that way they can just update the export and it will work for all subsequent `curl` commands,